storage/object/reference-content/cname-record.mdx

```diff
@@ -0,0 +1,84 @@
+
+A CNAME record may look like the following:
+
+| Hostname / Alias           | Destination / Canonical Name  |
+|----------------------------|-------------------------------|
+| `videos.example.com`       | `otherdomain.com`             |
+
+In this case, when a DNS server sees this record for `videos.example.com` it will know not to direct traffic to `blog.example.com`'s own IP address, but to that of `otherdomain.com`. It will find `othercomain.com`'s IP address via its [A record](/network/domains-and-dns/concepts/#resource-records).
+
+When the client actually connects to `otherdomain.com`'s IP address, the web server can see that the requested URL was `videos.example.com`, and deliver the relevant content.
+
+## When and why do I need to create a CNAME record for Edge Services?
+
+When you enable Edge Services, initially your bucket content is served through the standard Edge Services endpoint, e.g. `bucket-name.edge.scw.cloud`. If you do not want to customize the standard Edge Services endpoint, you do not need to worry about CNAME records.
+
+However, if you choose to [customize your Edge Services endpoint with your own subdomain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain), you are prompted, as part of the process, to create a CNAME record to point your subdomain to the Edge Services endpoint for your bucket. 
+
+    TODO SCREENSHOT
+
+## How to create a CNAME record
+
+Log into your domain provider, and locate the DNS settings for your domain. Create a new CNAME record pointing your subdomain to the Edge Services endpoint for your bucket. This endpoint will be displayed in the Scaleway console.
+
+The interface used by different domain providers varies, but creating your CNAME record may look like one of the following examples:
+
+| Record                                     | Destination                              |
+|--------------------------------------------|------------------------------------------|
+| `my-chosen-subdomain.beautiful-domain.com` | `bucket-name.edge.scw.cloud`             |
+
+| Subdomain                                  | Target host                                    |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
+
+| Host record                                | Points to                                      |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
```

Comment:
@RoRoJ the correct target is `bucket-name.svc.edge.scw.cloud.`
I believe you need to add the trailing . (at least in Scaleway DNS product) if you want the DNS to properly interpret the destination - to check in other DNS providers.